packages/@ourworldindata/core-table/src/OwidTable.ts

```diff
@@ -606,11 +606,30 @@ export class OwidTable extends CoreTable<OwidRow, OwidColumnDef> {
     }
 
     // Give our users a clean CSV of each Grapher. Assumes an Owid Table with entityName.
-    toPrettyCsv(useShortNames: boolean = false): string {
+    toPrettyCsv(
+        //
+        useShortNames: boolean = false,
+        activeColumnSlugs: string[] | undefined = undefined
+    ): string {
+        const activeColumnsDiff: Set<string> = activeColumnSlugs
+            ? differenceOfSets([
+                  new Set(this.columnSlugs),
+                  new Set(activeColumnSlugs),
+              ])
+            : new Set()
+        const columnSlugsToRemove = differenceOfSets([
+            activeColumnsDiff,
+            new Set([
+                OwidTableSlugs.year,
+                OwidTableSlugs.day,
+                this.entityNameSlug,
+            ]),
+        ])
```

Comment:
I think this all would be a bit cleaner and easier to follow if it was using `this.select()` instead. The two diffs after one another are hard to reason about, after all.

I.e. something like:
```ts
if (activeColumnSlugs?.length) {
	this.select([year, day, entityName, ...activeColumnSlugs])
} else {
	this.dropColumns(...)
}
```

Reply:
Ah very nice! I looked a bit to see if we had the inverse of drop and was looking for pick or keep but didn't think of select. Much nicer, thanks!